.github/mergify.yml

```diff
@@ -22,6 +22,7 @@ pull_request_rules:
           - branch-5.2
         assignees:
           - "{{ author }}"
+        bot_account: "{{ author }}"
```

Comment:
If Mergify provides similar parm for the merging-user it will be better

Reply:
What do you mean by the merging user? in the original PR?

Reply:
I discarding this PR, but just incase we will need it or something we can also use `bot_account: "{{ merged_by }}"`